Makefile

```diff
@@ -29,25 +29,18 @@ clean:
 services: init install
 	$(MAKE) -C services/nginz
 
-.PHONY: integration
-integration: fast
-	# We run "i" instead of "integration" to avoid useless rebuilds
-	# (since after "fast" everything will be built already)
+.PHONY: i
+i:
 	$(MAKE) -C services/cargohold i
 	$(MAKE) -C services/galley i
 	$(MAKE) -C services/brig i
 	$(MAKE) -C services/gundeck i-fake-aws
 	$(MAKE) -C services/spar i
 
+integration: fast i
+
 .PHONY: integration-aws
 integration-aws: fast
-	# We run "i" instead of "integration" to avoid useless rebuilds
```

Comment:
Did you also want to add running I-aws here?

Reply:
> Did you also want to add running I-aws here?

good idea.  especially since the first commit is not just half-baked, but plain wrong.  thanks!